todo_app/app.py

```diff
@@ -1,11 +1,24 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
 
 from todo_app.flask_config import Config
 
+from .data import session_items
+
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session_items.get_items()
+    return render_template("index.html", items=items)
+
+@app.post('/item')
+def add_item():
+    print(request.__dict__)
```

Comment:
Really useful for debugging, but worth noting that this could easily end up being a GDPR nightmare in production - any personal data that users send to you in their request could now end up in your logs which typically have much looser access than your production database! Not a real issue for us here, but worth considering